x/committee/keeper/invariants.go

```diff
@@ -71,17 +71,12 @@ func ValidProposalsInvariant(k Keeper) sdk.Invariant {
 				}
 			}
 
-			com, found := k.GetCommittee(ctx, proposal.CommitteeID)
+			_, found := k.GetCommittee(ctx, proposal.CommitteeID)
```

Comment:
nit, I don't think this invariant is extremely valuable since the proposals and votes are validated when someone submits them